lib/hyrax/configuration.rb

```diff
@@ -561,6 +561,8 @@ def translate_id_to_uri
 
     attr_writer :resource_id_to_uri_transformer
     def resource_id_to_uri_transformer
+      Deprecation.warn('Use Hyrax.config.translate_uri_to_id instead.')
+
```

Comment:
The deprecation statement seems to have the direction of the translation backward.  Should be...
```
Deprecation.warn('Use Hyrax.config.translate_id_to_uri instead.')
```

As another note about this.  The code here was written to remove references to ActiveFedora.  It is an equivalent set of code to what is done in ActiveFedora.  So yes to the deprecation, but `Hyrax.config.translate_id_to_uri` should probably be updated to this code to remove references to ActiveFedora.

Reply:
Any changes here should also be reflected in lib/generators/hyrax/templates/config/initializers/hyrax.rb

Reply:
Probably need similar changes for `translate_uri_to_id`.

Reply:
ah, i'm following this whole line better after these comments. thanks for this context.

question: since this is configuration for ActiveFedora, and for the Wings storage adapter that is designed for AF compatibility, is it okay to just leave this as-is and remove these methods entirely when Wings is removed?